services/ui-src/src/components/cards/ObjectiveProgressEntity.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { render, screen } from "@testing-library/react";
+import { mockObjectiveProgressFormattedEntityData } from "utils/testing/setupJest";
+import { ObjectiveProgressEntity } from "./ObjectiveProgressEntity";
+import { testA11y } from "utils/testing/commonTests";
+
+const objectiveProgressEntityComponent = (
+  <ObjectiveProgressEntity
+    formattedEntityData={mockObjectiveProgressFormattedEntityData}
+    entityCompleted={true}
+  />
+);
+
+describe("<ObjectiveProgressEntity />", () => {
+  beforeEach(() => {
+    render(objectiveProgressEntityComponent);
+  });
+  test("ObjectiveProgressEntity renders correctly", () => {
+    expect(
+      screen.getByRole("heading", { name: "mockObjectiveName" })
+    ).toBeVisible();
+  });
+
+  test("ObjectiveProgressEntity renders evalutionStep correctly", () => {
+    expect(
+      screen.getByText(
+        `${mockObjectiveProgressFormattedEntityData.quarterProjections[0].id} Target:`
+      )
+    ).toBeVisible();
+    expect(
+      screen.getByText(
+        `${mockObjectiveProgressFormattedEntityData.quarterProjections[1].id} Target:`
+      )
+    ).toBeVisible();
+    expect(
+      screen.getByText(
+        `${mockObjectiveProgressFormattedEntityData.quarterActuals[0].id} Actual:`
+      )
+    ).toBeVisible();
+    expect(
+      screen.getByText(
+        `${mockObjectiveProgressFormattedEntityData.quarterActuals[1].id} Actual:`
+      )
+    ).toBeVisible();
+    expect(
+      screen.getByText(
+        mockObjectiveProgressFormattedEntityData.performanceMeasureProgress
+      )
+    ).toBeVisible();
+    expect(
+      screen.getByText(mockObjectiveProgressFormattedEntityData.targetsMet)
+    ).toBeVisible();
+    expect(
+      screen.getByText(
+        mockObjectiveProgressFormattedEntityData.missedTargetReason
+      )
+    ).toBeVisible();
+  });
```

Comment:
This can one test block with `render(objectiveProgressEntityComponent);` inside it. (So it's isolated from `testA11y`.)

You could also simplify this with a loop, something like:
```
[ text1, text2, text3 ].forEach((text) => expect(screen.getByText(text)).toBeVisible())
```